util/time.go

```diff
@@ -24,6 +26,261 @@ const (
 	UnixTimeUnitOffset = uint64(time.Millisecond / time.Nanosecond)
 )
 
+var (
+	_ Clock = &RealClock{}
+	_ Clock = &MockClock{}
+
+	_ Ticker = &RealTicker{}
+	_ Ticker = &MockTicker{}
+
+	_ TickerCreator = &RealTickerCreator{}
+	_ TickerCreator = &MockTickerCreator{}
+)
+
+var (
+	realClock    *RealClock
+	currentClock *atomic.Value
+
+	realTickerCreator    *RealTickerCreator
```

Comment:
Maybe this field is unnecessary?

Reply:
Now it is used to do a safe fallback like in `CurrentTickerCreator`.